editor/src/components/shared/project-components.ts

```diff
@@ -352,8 +352,8 @@ const conditionalElementsDescriptors: ComponentDescriptorsForFile = {
         elementToInsert: jsxConditionalExpressionWithoutUID(
           jsExpressionValue(true, emptyComments),
           'true',
-          jsExpressionValue(null, emptyComments),
-          jsExpressionValue(null, emptyComments),
+          jsExpressionValue('True branch', emptyComments),
+          jsExpressionValue('False branch', emptyComments),
```

Comment:
Was this intended to be part of this PR, or is this PR branching off another piece of work?